pkg/volume/volume.go

```diff
@@ -28,6 +28,22 @@ import (
 type Volume interface {
 	// GetPath returns the directory path the volume is mounted to.
 	GetPath() string
+
+	Accountable
+}
+
+type VolumeDefaults struct {
+	AccountingNil
+}
+
+type Accountable interface {
+	GetAccounting() (*Accounting, error)
+}
+
+const unknownSize = -1
```

Comment:
Yeah I am glad you commented on this.  Is 0 fine for uninitialized, or will we need to ever return 0 as an actual value and differentiate.